tools/internal_ci/linux/grpc_bazel_build_in_docker.sh

```diff
@@ -24,4 +24,4 @@ git clone /var/local/jenkins/grpc /var/local/git/grpc
 && git submodule update --init --reference /var/local/jenkins/grpc/${name} \
 ${name}')
 cd /var/local/git/grpc
-bazel build --spawn_strategy=standalone --genrule_strategy=standalone :all test/... examples/...
+bazel build --spawn_strategy=standalone --genrule_strategy=standalone :all test/...
```

Comment:
I'm fine with removing these for now, but can we put a `TODO` here to add them back?